vagrant/provisioning/roles/arkcase-prerequisites/tasks/main.yml

```diff
@@ -55,27 +55,37 @@
       vars:
         item: arkcase
 
+- name: "Check if node is installed"
+  become: yes
+  shell: "node --version"
+  register: is_present
+  ignore_errors: true
+
 - name: ensure the nvm install script is downloaded
   become: yes
   shell: "curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash"
+  when: is_present.stdout == ""
 
 - name: install node, npm and set version
   become: yes
   become_user: root
   shell: "source /root/.nvm/nvm.sh && nvm install 6.17.1 && nvm alias default v6.17.1"
   args:
     executable: /bin/bash
+  when: is_present.stdout == ""
 
 - name: install the module 'yarn' with npm required by ArkCase
   become: yes
   npm:
     name: yarn
     global: yes
     state: present
+  when: is_present.stdout == ""
```

Comment:
Just because node was not installed previously, doesn't mean yarn was also not installed, but maybe it doesn't hurt to install yarn again even if it is already there.